onnxruntime/core/providers/tensorrt/tensorrt_execution_provider_helper.cc

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/shared_library/provider_api.h"
+#include "tensorrt_execution_provider.h"
+#include <iostream>
+
+namespace onnxruntime {
+
+// The newly-built graph has not yet being resolved by Graph::Resolve(), so we can't leverage ORT Graph IsInputInitializerOrOutput() API.
+// We have to do it by ourselves.
+bool TensorrtExecutionProvider::IsInputInitializerOrOutput(Graph* graph,
+                                                           const std::string& name,
+                                                           bool check_ancestors,
+                                                           std::unordered_map<std::string, std::unique_ptr<SubGraphContext>>& subgraph_context_map) const {
```

Comment:
nit: please keep lines to max length 120.

most editors have the ability to display a vertical ruler at 120 lines (e.g. Editor Guidelines for VS, VS Code has the 'editor.rulers' setting) so you can easily see which ones are too long.

Reply:
Made them less than length 120. Some lines can't be changed due to `lintrunner -a` changes them back.

Reply:
Which lines in particular? 

I typically have clang-format set to format on save so I can see what changes it will apply. Sometimes it's a case of needing a newline in a specific place or a training comma and format-on-save lets me figure out what it's looking for.

Reply:
The ones with `LOGS_DEFAULT(VERBOSE)  << ...`
For example,  in tensorrt_execution_provider_helper:138
`LOGS_DEFAULT(VERBOSE) << "[TensorRT EP] \t" << n_input.Name() << " is added as an explicit input into the newly built graph";`, I added newline right after `<<`, but will be changed back by lintrunner. Not yet figuring out where to insert the newline.

Reply:
Weird. lintrunner doesn't alter that line for me. I'd rather leave the long line than make it less efficient by using '+'. Maybe ask Justin what is doing the modification as I get different behavior.